packages/atlas/src/components/WelcomeView/WelcomeView.styles.ts

```diff
@@ -158,6 +158,11 @@ export const ImageGridItem = styled(GridItem)`
   ${media.md} {
     flex-direction: column-reverse;
     height: calc(100vh - 80px);
+    transform: scale(1.2);
```

Comment:
I wanted to check how this look but I can't signin rn. It's just that scaled element can sometime look a bit weird.

Reply:
#5464